libraries/plugins/witness/witness.cpp

```diff
@@ -74,6 +77,8 @@ void witness_plugin::plugin_set_program_options(
           DEFAULT_VALUE_VECTOR(std::make_pair(chain::public_key_type(default_priv_key.get_public_key()),
                 graphene::utilities::key_to_wif(default_priv_key))),
                 "Tuple of [PublicKey, WIF private key] (may specify multiple times)")
+         ("private-key-file", bpo::value<vector<boost::filesystem::path>>()->composing()->multitoken(),
+          "Path to file containing tuples of [PublicKey, WIF private key] (may specify multiple times)")
```

Comment:
AFAICS the implementation currently allows only one such pair per file. Or did I miss something?

Reply:
Yes, multiple files, one key pair per file.

Reply:
Please either allow multiple pairs, or replace "tuples" above. (IMO allowing only one pair per file is counterintuitive, so the explanation text should stress that limitation.)

Reply:
A very good objection. I have modified the code to read one pair per line. Please, inspect it.

Reply:
Thanks. Please modify the explanation so the user understands that each line must contain exactly one such pair.

Reply:
Done.

// Welcome back from holiday, I hope. :)

Reply:
Thanks. :)